app/views/events/show.slim

```diff
@@ -1,24 +1,55 @@
-h2 = @event.name
-p Starts #{@event.starting_date}
-h3 In #{@event.location} | #{@event.hashtag} 
-b Description
-p #{@event.description}
-h3 Interests
-p
-  -@event.interest_list.each do |interest|
-    ="#{interest} |"
-
-p.fa.fa-3x.fa-user = "#{@event.attendees.size}"
--if current_user.nil?
-  p Wanna attend to this event?
-  = link_to 'Sign in with your twitter account', user_omniauth_authorize_path(:twitter)
-
--else
-  - if @event.attending?(current_user.id)
-    = link_to 'Attend', attend_event_path(@event), method: :post
-  - else
-    = link_to 'Cancel', cancel_attend_event_path(@event), method: :delete
-  - if @event.owner == current_user.id
-    = link_to 'Edit Event', edit_event_path(@event)
-    = link_to 'Delete Event', event_path(@event), method: :delete
+.BodyLayout
+  .BodyLayout-header
+    .Heading.Heading--large = @event.name
+  .BodyLayout-content
+    .Grid.Grid--threeColumns
+      .Grid-item
+        .InfoLayout
```

Comment:
This should also be merged with both form layouts. And you should give a name that works for the form and data displaying scenarios.